specification/batch/resource-manager/Microsoft.Batch/stable/2021-01-01/BatchManagement.json

```diff
@@ -0,0 +1,5332 @@
+          "readOnly": true,
+          "description": "The current state of the application package.",
+          "enum": [
+            "Pending",
+            "Active"
+          ],
+          "x-ms-enum": {
+            "name": "PackageState",
+            "modelAsString": false,
+            "values": [
+              {
+                "value": "Pending",
+                "description": "The application package has been created but has not yet been activated."
+              },
+              {
+                "value": "Active",
+                "description": "The application package is ready for use."
+              }
+            ]
+          }
+        },
+        "format": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The format of the application package, if the package is active."
+        },
+        "storageUrl": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The URL for the application package in Azure Storage."
+        },
+        "storageUrlExpiry": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The UTC time at which the Azure Storage URL will expire."
+        },
+        "lastActivationTime": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time at which the package was last activated, if the package is active."
+        }
+      },
+      "description": "Properties of an application package"
+    },
+    "ListApplicationsResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Application"
+          },
+          "description": "The list of applications."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "The result of performing list applications."
+    },
+    "ListApplicationPackagesResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationPackage"
+          },
+          "description": "The list of application packages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "The result of performing list application packages."
+    },
+    "BatchLocationQuota": {
+      "properties": {
+        "accountQuota": {
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true,
+          "description": "The number of Batch accounts that may be created under the subscription in the specified region."
+        }
+      },
+      "description": "Quotas associated with a Batch region for a particular subscription."
+    },
+    "Resource": {
```

Comment:
For a new api-version, supporting a new top level property `systemData` is recommended for tracked / proxy resources.

https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html

You can reference the one in common-type.
https://github.com/Azure/azure-rest-api-specs/blob/a16488bbd0d17551c729e22ea6ac6f7c6588a0a3/specification/common-types/resource-management/v2/types.json#L445

Reply:
Talking to the service team, it looks like this would require changes on the service side to support properly. This seems like something we should work on supporting, but doesn't seem like it should block this API version update.

Reply:
@j5lim does that sound reasonable to you?